expansion/src/main/java/no/unit/nva/expansion/model/ExpandedGeneralSupportRequest.java

```diff
@@ -137,4 +139,32 @@ private Instant getCreatedDate() {
     public void setCreatedDate(Instant createdDate) {
         this.createdDate = createdDate;
     }
-}
+
+    private static ExpandedTicketStatus getExpandedTicketStatus(GeneralSupportRequest generalSupportRequest) throws NotFoundException {
+        switch (generalSupportRequest.getStatus()) {
+            case PENDING:
+                return getNewTicketStatus(generalSupportRequest);
+            case COMPLETED:
+                return ExpandedTicketStatus.COMPLETED;
+            case CLOSED:
+                return ExpandedTicketStatus.CLOSED;
+            default:
+                throw new NotFoundException ("Invalid GeneralSupportRequest status " + generalSupportRequest.getStatus());
+        }
+    }
+
+    private static ExpandedTicketStatus getNewTicketStatus(GeneralSupportRequest generalSupportRequest) {
+        if (isNull(generalSupportRequest.getAssignee())) {
+            return ExpandedTicketStatus.NEW;
+        } else {
+            return ExpandedTicketStatus.PENDING;
+        }
+    }
+
+    private TicketStatus getTicketStatus() {
+        if (!this.getStatus().equals(ExpandedTicketStatus.NEW)) {
+            return TicketStatus.parse(this.getStatus().toString());
+        }
+        return null;
```

Comment:
If status(ExpandedTicketStatus) is NEW, then TicketStatus = PENDING

Reply:
On second thought, if toTicketEntry() is only used in testing, move the code to the test.
This so the production code shouldn't have functionality it doesn't need -> like decompressing after an expansion.